Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Build initial stage with all dependencies from base image
+FROM anmolnetwork/anmol-node-build-test AS build_base_stage
+
+# Second stage to build prod image
+FROM alpine:3.14.0
+WORKDIR /pkg
+COPY --from=build_base_stage . .
+CMD [ "./target/release/anmol"]
```

Comment:
I may need to look into arguments for this, if any are needed.

Reply:
There's also some remaining issues with this final image that I'm starting to look into.